Permission and Access Helper/force-app/main/default/classes/ComboboxDataWrapper.cls

```diff
@@ -0,0 +1,12 @@
+public with sharing class ComboboxDataWrapper {
```

Comment:
Add copyright comment for a file.

Reply:
Should it be done just for the Apex classes or the JS and HTMLs too ?